plugins/netskope/help.md

```diff
@@ -0,0 +1,556 @@
+# Description
+
+Update a File Hash List using Netskope
+
+# Key Features
+
+Identify key features of plugin.
+
+# Requirements
+
+* Example: Requires an API Key from the product
+* Example: API must be enabled on the Settings page in the product's user interface
```

Comment:
You can remove `example` here, so your requirements are simply:
* Requires an API Key from the product
* API must be enabled on the Settings page in the product's user interface

Reply:
You're right! I'll fix that.